### PR TITLE
Edit icons: 3x3/OH/BLD Relay and Team Relay, 3x3 BLD 4/8-man Relay and Mini Guildford

### PR DESCRIPTION
**Pull request by dmint789**
This PR edits the following icons:

1. Mini Guildford
2. 3x3 BLD 4-man Relay
3. 3x3 BLD 8-man Relay
4. 3x3 + OH + BLD Relay
5. 3x3 + OH + BLD Team Relay

All changes except Mini Guildford are simple tweaks based on actually using the new icons. Mini Guildford includes the following changes:

- Reposition the elements to respect padding
- Make the OH event consistent with the 333oh icon
- Use the Clock's handle, since it doesn't add too much detail, while making it clear this is the Clock event

 Preview:

minigf

### CODE REVIEW

File: svgs/unofficial/333_oh_bld_relay.svg
Comment (jfly) [inline svgs/unofficial/333_oh_bld_relay.svg:1]: Why did you change the order of the events here? It feels reasonable to me that this would read in reading order: 333, 333oh, 333bf. (same question in other places)

Comment (dmint789) [reply svgs/unofficial/333_oh_bld_relay.svg:1]: Typically, things are read top to bottom, left to right. So since this event is referred to as 3x3 + OH + BLD, I put 3x3 first, then OH, then BLD. I shouldn't have made the mistake in the first place, really.

Comment (jfly) [reply svgs/unofficial/333_oh_bld_relay.svg:1]: I personally read things left to right, top to bottom. I know Japenese goes top to bottom right to left, and Hebrew goes right to left, top to bottom, but I'm not familiar with anything that goes tops to bottom, left to right.

Comment (dmint789) [reply svgs/unofficial/333_oh_bld_relay.svg:1]: Well, I meant left to right, top to bottom. We start at row 1 and see a 3x3. Then the row ends, so we go to row 2, starting at OH. Then we continue down that row and see BLD. Same as pretty much all European languages.

Comment (jfly) [reply svgs/unofficial/333_oh_bld_relay.svg:1]: Ah whoops! I totally misread things earlier. Yes, this change makes sense. Sorry for the confused comment.

**Review by jfly — APPROVED** [review]

**Comment by dmint789** [comment]
Oops, I accidentally closed this while trying to merge (forgetting that I don't have the rights). @jfly can you merge this?

**Comment by dmint789** [comment]
@jfly could you also please deploy this? 🙏

**Comment by jfly** [comment]
@dmint789, we don't really deploy this anywhere. Are you asking for me to do a release?

**Comment by dmint789** [comment]
@jfly I meant release, yeah, my bad

**Comment by jfly** [comment]
@dmint789 v1.1.1 is released: https://www.npmjs.com/package/@cubing/icons/v/1.1.1
